primitives/manta/src/constants.rs

```diff
@@ -96,3 +100,21 @@ pub const MANTA_PAY_PALLET_ID: PalletId = PalletId(*b"mantapay");
 ///
 /// This should only be used for testing and should not be used in production.
 pub const TEST_DEFAULT_ASSET_ED: Balance = 1;
+
+/// 1_000_000_000_000
+pub const WEIGHT_PER_SECOND: Weight = 1_000_000_000_000;
+/// 1_000_000_000
+pub const WEIGHT_PER_MILLIS: Weight = WEIGHT_PER_SECOND / 1000;
+/// 1_000_000
+pub const WEIGHT_PER_MICROS: Weight = WEIGHT_PER_MILLIS / 1000;
+/// 1_000
+pub const WEIGHT_PER_NANOS: Weight = WEIGHT_PER_MICROS / 1000;
+
+parameter_types! {
+    /// By default, Substrate uses RocksDB, so this will be the weight used throughout
+    /// the runtime.
+    pub const RocksDbWeight: RuntimeDbWeight = RuntimeDbWeight {
```

Comment:
Because we're going to control all constants, but if there's any change from upstream, maybe we don't know it, so we better write some test cases to cover it, like
```rust
#[test]
fn diff_constants_with_upstream() {
    ...
    assert!(manta_primitives::RocksDbWeight == frame_support::weights::RuntimeDbWeight);
    ...
}
```
`RocksDbWeight` is a primitive constant, we should cover primitive constants only.

Reply:
I already moved `RocksDbWeight` to manta-primitives, because it is not just needed in the runtime/ folder.

Reply:
Not sure I get your point. I mean maybe we won't keep it up-to-date, we should know changes from upstream 's `RocksDbWeight`. So writing test case to compare both `RocksDbWeight` is a solution.

Reply:
I agree with Jamie here that there should be a test whether our `RocksDbWeight` constant is equal to the parity-provided `frame_support::weights:: RocksDbWeight` so we get pushed to investigate the change if they desync.

Reply:
Oooh I get what you guys mean now, yeah no problem.